calendar/Package.md

```diff
@@ -1,349 +1,20 @@
-[![Build](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/actions?query=workflow%3ACI)
-[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-googleapis.calendar.svg)](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/commits/master)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+Connects to Google Calendar from Ballerina
 
-# Ballerina Google Calendar Connector 
-Connects to Google Calendar using Ballerina.
+## Package overview
+The `ballerinax/googleapis.calendar` is a [Ballerina](https://ballerina.io/) connector for Google Calendar.
 
-# Introduction
-## Google Calendar
-[Google Calendar](https://developers.google.com/calendar) is a time-management and scheduling calendar service developed by Google. It lets users to organize their schedule and share events with others. The Google Calendar endpoint allows you to access the Google Calendar API Version v3 through Ballerina.
+This package provides the capability to manage calendars and events in the Google Calendar and listen the events occurred in the calendar. 
```

Comment:
listen to events

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884